app_template_iframe/assets/iframe.html

```diff
@@ -1,16 +1,21 @@
+<!doctype html>
 <html>
 <head>
   <meta charset="utf-8">
-  <!-- http://garden.zendesk.com -->
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0" type="text/css">
+  <!--   See Using Zendesk Garden:
+    https://developer.zendesk.com/apps/docs/developer-guide/setup#using-zendesk-garden
+    https://garden.zendesk.com/css-components/bedrock/
+    https://garden.zendesk.com/css-components/utilities/typography/
+   -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0.20/dist/index.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-utilities@4.2.1/dist/index.min.css">
```

Comment:
@pmgrove recently mentioned to me that we should use `bedrock` for `zat new`, not sure about `utilities` though. cc @sostopher and @ocke for thoughts too.

Reply:
I added Bedrock back in but kept Utilities, so `u-semibold u-fs-xl` could be used.

Reply:
You can make this 1 request (creating the connection is the slowest part of downloading an app).

`<link rel="stylesheet" href="https://cdn.jsdelivr.net/combine/npm/@zendeskgarden/css-bedrock@7.0.20,npm/@zendeskgarden/css-utilities@4.2.1">`